haystack/nodes/reader/table.py

```diff
@@ -241,76 +243,69 @@ def __init__(
             model_name_or_path, revision=model_version, use_auth_token=use_auth_token
         )
         if tokenizer is None:
-            self.tokenizer = TapasTokenizer.from_pretrained(model_name_or_path, use_auth_token=use_auth_token)
+            self.tokenizer = TapasTokenizer.from_pretrained(
+                model_name_or_path, use_auth_token=use_auth_token, model_max_length=max_seq_len
```

Comment:
Pass the `max_seq_len` to the loading of the tokenizer to allow the user to change this value.